### DISCUSSION TITLE
FR: Add Open Location Codes (aka "Plus Codes") support

### DISCUSSION THREAD

**Discussion by pataquets** [minad/osm#47] [Ideas]
[Open Location Codes](https://en.wikipedia.org/wiki/Open_Location_Code) are very handy for encoding and sharing location infos.

Support could be broken into two features:
- Accept/decode OLCs as "go to" coordinates.
- Generating/encoding map locations as OLCs.

EDIT: there is some lib here: [ralph-schleicher/open-location-code: An Open Location Code library for Common Lisp.](https://github.com/ralph-schleicher/open-location-code)

**Comment by minad** [comment]
Sounds good, patches welcome.

**Reply by pataquets** [reply]
Thanks for considering.
Edited OP to add relevant lib.

**Reply by minad** [reply]
> Edited OP to add relevant lib.

I don't think this lib is of any use, since it is written in CL. Directly porting it is not possible since this would conflict with FSF copyright assignment.
